app/controllers/pages_controller.rb

```diff
@@ -18,11 +18,17 @@ def contact
   end
 
   def send_message
-    send_email_mailgun
+    if send_email_mailgun
```

Comment:
Will this always return something truthy? Ever return anything falsey? (nil, false)

Reply:
I'll change it back